tests/test_app/TestApp/Template/FooPrefix/BarPrefix/Element/prefix_element.ctp

```diff
@@ -0,0 +1 @@
+<?= 'this is a nested prefixed test element'; ?>
```

Comment:
No particular reason.
I duplicated a flash element file from the TestPlugin provided with the tests suite and just changed the output ([link to file](https://github.com/cakephp/cakephp/blob/3.1/tests/test_app/Plugin/TestPlugin/src/Template/Element/Flash/plugin_element.ctp)).
I'll change it to make it more "real life example".

And ok for new lines, I'll add them.